src/pseudopeople/interface.py

```diff
@@ -103,16 +104,16 @@ def _generate_dataset(
 
 def validate_source_compatibility(source: Path):
     # TODO [MIC-4546]: Clean this up w/ metadata and update test_interface.py tests to be generic
-    changelog = source / "CHANGELOG.rst"
-    if changelog.exists():
-        version = _get_data_changelog_version(changelog)
-        if version > parse("1.4.2"):
+    metadata = source / "metadata.yaml"
+    if metadata.exists():
```

Comment:
Won't this do nothing if the user tries to run it on the old data, which doesn't have a metadata file?

Reply:
It raises an error if there isn't one.